Gruntfile.js

```diff
@@ -181,9 +181,33 @@ module.exports = function (grunt) {
 	];
 
 	grunt.initConfig(gruntConfig);
-
 	grunt.registerTask('compiler', ['curl:compiler', 'unzip:compiler']);
-	grunt.registerTask('all', ['copy', 'concat', 'closure-compiler']);
+	grunt.registerTask('all', ['copy', 'concat', 'requirejs', 'closure-compiler']);
+
+	grunt.loadTasks('spec/tasks');
+
+	var browsers = grunt.option('browsers') || 'default';
+  grunt.registerTask('test',
```

Comment:
I do, but right now I just have bigger things to worry about than to fix an issue that is only visible in Github because it assumes tabs are 4 spaces instead of 2. The rest of the library uses spaces, so this file is the exception. I'll do a fix up, but I am more concerned with the task in hand to be honest which is to get the Javascript library running tests automaticalky in a range of browsers. 

Regards,
Matthew

Sent from my phone

> On 23 Feb 2015, at 09:40, Vincent Bonmalais <notifications@github.com> wrote:
> 
> In Gruntfile.js:
> 
> >  	grunt.registerTask('compiler', ['curl:compiler', 'unzip:compiler']);
> > -	grunt.registerTask('all', ['copy', 'concat', 'closure-compiler']);
> > +	grunt.registerTask('all', ['copy', 'concat', 'requirejs', 'closure-compiler']);
> > +
> > +	grunt.loadTasks('spec/tasks');
> > +
> > +	var browsers = grunt.option('browsers') || 'default';
> > +  grunt.registerTask('test',
> Mixing tabs and spaces? You are kidding, right ? You are pretty good to keep a consistent style usually. I didn't expect you wouldn't care about this.
> 
> —
> Reply to this email directly or view it on GitHub.
>